client/my-sites/checkout/checkout/stripe-elements-payment-box.jsx

```diff
@@ -0,0 +1,268 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { loadScript } from '@automattic/load-script';
+import { overSome } from 'lodash';
+import page from 'page';
+import debugFactory from 'debug';
+import React, { useEffect, useState } from 'react';
+import { localize } from 'i18n-calypso';
+import { StripeProvider, Elements, injectStripe, CardElement } from 'react-stripe-elements';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import notices from 'notices';
+import wpcom from 'lib/wp';
+import { paymentMethodClassName, getLocationOrigin } from 'lib/cart-values';
+import CheckoutTerms from './checkout-terms';
+import { isWpComBusinessPlan, isWpComEcommercePlan } from 'lib/plans';
+import PaymentChatButton from './payment-chat-button';
+import SubscriptionText from './subscription-text';
+import PaymentCountrySelect from 'components/payment-country-select';
```

Comment:
I'm wondering if we could drop this:
> The card Element automatically determines your customer’s billing address country based on their card number. 

Maybe we can fill the country based on the card?

Reply:
I think we should definitely do that. I'll just have to figure out how 😁